blivet/populator/populator.py

```diff
@@ -118,7 +118,7 @@ def _udev_device_is_disk(self, info):
             udev/sysfs, we have to define what is a disk in terms of what is
             not a disk.
         """
-        return udev.device_is_realdisk(info)
+        return udev.device_is_disk(info)
```

Comment:
You are right. Deprecation warning popping out is targeted to version 3.0. This part of code is to be left unchanged.